# Determination - Deterministic rendering environment for white-axe's music
# Copyright (C) 2024 Liu Hao <whiteaxe@tuta.io>
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.

# This patch disables the `loadPart()` function because it doesn't work anyways.

--- a/src/Misc/MiddleWare.cpp
+++ b/src/Misc/MiddleWare.cpp
@@ -610,7 +610,7 @@ public:
     }
 
     void loadPart(int npart, const char *filename, Master *master, rtosc::RtData &d)
-    {
+    { return;
         actual_load[npart]++;
 
         if(actual_load[npart] != pending_load[npart])
